Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -740,17 +905,14 @@ BTDiagnostics::DefineFieldBufferMultiFab (const int i_buffer, const int lev)
             m_geom_output[i_buffer][lev] = amrex::refine( m_geom_output[i_buffer][lev-1],
                                                           warpx.RefRatio(lev-1) );
         }
-    }
+//    }
 }
 
 
 void
 BTDiagnostics::DefineSnapshotGeometry (const int i_buffer, const int lev)
 {
-    if ( m_do_back_transformed_fields ) {
         auto & warpx = WarpX::GetInstance();
```

Comment:
alignment of this scope is now off due to the removed `if` - feel free to update :)